CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Change Log
 
+## [0.22.0](https://github.com/eclipse/lemminx/milestone/35?closed=1) (September 28, 2022)
+
+### Enhancements
+
+ * Suppress validation kinds based on file pattern. See [#1275](https://github.com/eclipse/lemminx/issues/1275).
+ * Fix CI badge, add some more badges. See [#1262](https://github.com/eclipse/lemminx/pull/1262).
```

Comment:
I would probably move this entry near the bottom, to try to prioritize the enhancements that are more interesting to a user.